tests/operator/suite.go

```diff
@@ -128,6 +129,75 @@ func LoadChecks() {
+		WithCheckFn(func(c *checksdb.Check) error {
+			testOnlySingleNamespacedOperatorsAllowedInTenantNamespaces(c, &env)
+			return nil
+		}))
+}
+
+// This function checks if SingleNamespaced Operators should only be installed in the tenant dedicated operator namespace
+func testOnlySingleNamespacedOperatorsAllowedInTenantNamespaces(check *checksdb.Check, env *provider.TestEnvironment) {
+	check.LogInfo("Starting testInstalledSingleNamespaceOperatorInTenantNamespace")
+
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	operatorNamespaces := make(map[string]bool)
+	for _, operator := range env.Operators {
+		operatorNamespace := operator.Csv.Annotations["olm.operatorNamespace"]
+		for _, namespace := range env.Namespaces {
+			if namespace == operatorNamespace {
+				operatorNamespaces[operatorNamespace] = true
+			}
+		}
+	}
+
+	for operatorNamespace := range operatorNamespaces { // operator installed namespace
+		check.LogInfo("Checking if namespace %s contains only valid single/ multi namespaced operators", operatorNamespace)
+
+		isDedicatedOperatorNamespace, singleOrMultiNamespaceOperators, nonSingleOrMultiNamespaceOperators,
+			csvsFoundButNotInOperatorInstallationNamespace, operatorsFoundButNotUnderTest, podsNotBelongingToOperators := checkValidOperatorInstallation(operatorNamespace)
+
+		check.LogInfo("isDedicatedOperatorNamespace=%t, singleOrMultiNamespaceOperators=%s, nonSingleOrMultiNamespaceOperators=%s, csvsFoundButNotInOperatorInstallationNamespace=%s, operatorsFoundButNotUnderTest=%s, podsNotBelongingToOperators=%s", isDedicatedOperatorNamespace, singleOrMultiNamespaceOperators, nonSingleOrMultiNamespaceOperators, csvsFoundButNotInOperatorInstallationNamespace, operatorsFoundButNotUnderTest, podsNotBelongingToOperators)
+
+		if isDedicatedOperatorNamespace {
+			var msg string
+			if len(singleOrMultiNamespaceOperators) == 0 {
+				msg = "Namespace contains no installed single/multi namespace operators"
+			} else {
+				msg = fmt.Sprintf("Namespace is dedicated to single/multi namespace operators %s ", strings.Join(singleOrMultiNamespaceOperators, ", "))
+			}
+
+			compliantObjects = append(compliantObjects, testhelper.NewNamespacedReportObject(msg, testhelper.Namespace, true, operatorNamespace))
+		} else {
+			msg := "Namespace is a non-compliant operator namespace with "
+
+			if len(nonSingleOrMultiNamespaceOperators) != 0 {
+				msg += " - invalid operators " + strings.Join(nonSingleOrMultiNamespaceOperators, ", ")
+			}
+
+			if len(nonSingleOrMultiNamespaceOperators) != 0 {
+				msg += "- invalid installed operators (not single/multiple namespaced) " + strings.Join(nonSingleOrMultiNamespaceOperators, ", ")
+			}
+			if len(csvsFoundButNotInOperatorInstallationNamespace) != 0 {
+				msg += "- invalid installed operators (installed in other namespace) " + strings.Join(csvsFoundButNotInOperatorInstallationNamespace, ", ")
```

Comment:
msg += "- invalid installed operators (installed in other namespace) " + strings.Join(csvsFoundButNotInOperatorInstallationNamespace, ", ")
->
msg += "- this namespace is the target namespace of other operators (" + strings.Join(csvsFoundButNotInOperatorInstallationNamespace, ", ")+")\n"